app/scripts/models/notifications.js

```diff
@@ -13,7 +13,8 @@ define([
   'use strict';
 
   var EVENTS = {
-    PROFILE_CHANGE: 'profile:change'
+    PROFILE_CHANGE: 'profile:change',
+    LOGOUT: 'fxaccounts:logout'
```

Comment:
We should probably think about how these namespaces are organized. Given that we have `profile:change`, it seems "fxaccounts" is already implied. It looks a little funny as the prefix in the logout event.